internal/testrunner/runners/system/runner.go

```diff
@@ -1333,6 +1376,18 @@ func (r *runner) validateTestScenario(ctx context.Context, result *testrunner.Re
 		return result.WithError(err)
 	}
 
+	// TODO: remove this once Elasticsearch will map event.original correctly (https://github.com/elastic/elasticsearch/pull/106714)
```

Comment:
Btw, if we remove this, we won't be checking ignored fields, right? :thinking: Or we will have to add a different way to check ignored fields?

Reply:
I just meant removing the exception for event.original and failing on any ignored field. But I agree with checking the Elasticsearch version is the better approach.

Reply:
Oh ok, I see. Then yes, we can conditionally omit `event.original` depending on the stack version.